src/app/api/admin/dashboard-metrics/route.ts

```diff
@@ -0,0 +1,57 @@
+// src/app/api/admin/dashboard-metrics/route.ts
+import { NextResponse } from "next/server";
+
+import { prisma } from "@/lib/prisma";
+
+export interface Metrics {
+  totalSales: number;
+  transactionCount: number;
+  activeUsers: number;
+  newUsersThisWeek: number;
+  averageOrderValue: number;
+  bookCount: number;
+}
+
+export async function GET() {
+  try {
+    const [totalSalesAgg, transactionCount, activeUsers, newUsersThisWeek] = await Promise.all([
+      prisma.transaction.aggregate({
```

Comment:
You need to enforce admin check here (using `getServerSession(authOptions)` for security as pointed by the AI